cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -183,6 +183,10 @@ private void sanityCheckOffsetFetch(Map<TopicPartition, Long> endOffsets,
    */
   private boolean refreshPartitionAssignment() {
     List<PartitionInfo> remotePartitionInfo = _metricConsumer.partitionsFor(_metricReporterTopic);
+    if (remotePartitionInfo == null) {
+      LOG.error(String.format("_metricConsumer returned null for _metricReporterTopic %s", _metricReporterTopic));
```

Comment:
Nit -> `LOG.error("_metricConsumer returned null for _metricReporterTopic {}", _metricReporterTopic);`

Reply:
I agree that this is a much cleaner expression of the same idea; see 615c6fa